base/math.jl

```diff
@@ -303,7 +303,8 @@ Stacktrace:
 ```
 
 !!! note
-    If `b` is a power of 2 or 10, [`log2`](@ref) or [`log10`](@ref) should be used, as these will
+    If `b` is a power of 2 or 10, [`
```

Comment:
Looks like the `log2` reference got accidentally axed here.

Reply:
Oops